pkg/azuredisk/device_perf_optimization_helper.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azuredisk
+
+import (
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	PerfTuningModeNone       = "none"
+	PerfTuningModeAuto       = "auto"
+	PerfProfileDefault       = "default"
+	PremiumAccountPrefix     = "premium"
+	StandardSsdAccountPrefix = "standardssd"
+)
+
+// IsValidPerfTuningMode Checks to see if perf tuning mode passed is correct
+// Right now we only support auto perf tuning mode
+// Manual mode to come later
+func IsValidPerfTuningMode(mode string) bool {
```

Comment:
for these non public exposure functions, naming as `isValidPerfTuningMode` is ok, no need to expose as public function for every function, and add comment for every function